src/index.html

```diff
@@ -12,6 +12,14 @@
       <link rel="stylesheet" type="text/css" href="/styles/noscript.css" />
     </noscript>
 
+    <meta name="defaultLanguage" content="en-US">
+    <meta name="availableLanguages" content="en-US, pl">
+    <link rel="localization" href="/locale/{locale}/dictionary.ftl">
+
+    <script src="/scripts/browser.js"></script>
```

Comment:
I think so. It is probably good idea to have it all compiled together. I'll try to make it works with Grunt and building process.